sdk/identity/identity/Troubleshooting.md

```diff
@@ -201,48 +282,59 @@ The supported clouds are:
 
-The `AzureCliCredential` utilizes the currently logged in Azure user in Azure CLI to fetch an access token. You need to log in to your account in Azure CLI via the `az login` command. For further instructions, see [Sign in with Azure CLI](https://docs.microsoft.com/cli/azure/authenticate-azure-cli). Once logged in, try running the credential again.
+#### Verify the Azure CLI can obtain tokens
 
-## Troubleshoot Azure PowerShell authentication issues
+You can manually verify that the Azure CLI is properly authenticated and can obtain tokens. First, use the `account` command to verify the account which is currently logged into the Azure CLI. 
 
-### Credential unavailable error
+```azurecli
+az account show
+```
 
-#### PowerShell not installed
+Once you've verified the Azure CLI is using the correct account, you can validate it's able to obtain tokens for this account.
 
-The `Azure PowerShell Credential` utilizes the locally installed `PowerShell` command line tool to fetch an access token. Ensure it's installed on your platform by following the instructions [here](https://docs.microsoft.com/powershell/scripting/install/installing-powershell?view=powershell-7.1). Then run the credential again.
+```azurecli
+az account get-access-token --output json --resource https://management.core.windows.net
+```
+>Note that output of this command will contain a valid access token, and SHOULD NOT BE SHARED to avoid compromising account security.
 
-#### Azure Az Moudle not installed
+## Troubleshoot Azure PowerShell authentication issues
 
-The Azure PowerShell credential failed to execute, as the Azure Az module isn't installed. To use the Azure PowerShell credential, install the Azure Az PowerShell module:
+### CredentialUnavailableError
 
-```powershell
+| Error Message |Description| Mitigation |
+|---|---|---|
+|PowerShell isn't installed.|No local installation of PowerShell was found.|Ensure that PowerShell is properly installed on the machine. Instructions for installing PowerShell can be found [here](https://docs.microsoft.com/powershell/scripting/install/installing-powershell).|
+|Az.Account module >= 2.2.0 isn’t installed.|The `Az.Account` module needed for authentication in Azure PowerShell isn't installed.|Install the latest `Az.Account` module. Installation instructions can be found [here](https://docs.microsoft.com/powershell/azure/install-az-ps). To use the Azure PowerShell credential, install the Azure Az PowerShell module:```powershell
 Install-Module -Name Az -Scope CurrentUser -Repository PSGallery -Force
-```
+```|
```

Comment:
This

    ```powershell
    Install-Module -Name Az -Scope CurrentUser -Repository PSGallery -Force 
    ```

Is broken. I think the fix is to just inline it, like `Install-Module -Name Az -Scope CurrentUser -Repository PSGallery -Force`